src/test/java/org/elasticsearch/search/functionscore/ExplainableScriptTests.java

```diff
@@ -105,8 +108,12 @@ public double runAsDouble() {
         }
 
         @Override
-        public Explanation explain(float score) throws IOException {
-            return new Explanation((float) (runAsDouble()), "This script returned " + runAsDouble());
+        public Explanation explain(Explanation subQueryScore) throws IOException {
+            Explanation exp = new Explanation((float) (runAsDouble()), "This script returned " + runAsDouble());
+            Explanation _scoreExp = new Explanation(subQueryScore.getValue(), "_score: ");
```

Comment:
Can you remove the leading underscore? Variables should start with a lowercase letter according to http://www.oracle.com/technetwork/java/codeconventions-135099.html